decidim-initiatives/app/serializers/decidim/initiatives/open_data_initiative_serializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Initiatives
+    class OpenDataInitiativeSerializer < Decidim::Exporters::Serializer
+      # Serializes an initiative
+      def serialize
+        {
+          id: resource.id,
+          title: resource.title,
+          description: resource.description,
+          state: resource.state,
+          created_at: resource.created_at,
+          published_at: resource.published_at,
+          signature_end_date: resource.signature_end_date,
+          signature_type: resource.signature_type,
+          signatures: resource.supports_count,
+          scope: {
+            name: resource.scope&.name
+          },
+          type: {
+            title: resource.type&.title
```

Comment:
In Participatory process we export also the id. here we don't. Is this intended?